compose/ui/ui/src/skikoTest/kotlin/androidx/compose/ui/input/pointer/PointerIconTest.kt

```diff
@@ -286,12 +288,11 @@ class PointerIconTest {
         }
     }
 
+    @OptIn(ExperimentalTestApi::class)
```

Comment:
Should we use `runSkikoComposeUiTest` for other test cases in this file?

Reply:
Probably yes, but I'm planning to move all our testing to a commonized test framework at some point, and I'd rather not do it now just for this file.